config-default.yml

```diff
@@ -369,14 +371,18 @@ urls:
     paste_service:                      !JOIN [*SCHEMA, *PASTE, "/{key}"]
     site_logs_view:                     !JOIN [*STAFF, "/bot/logs"]
 
+    # Cloudflare workers
+    worker_domain: &WORKER_DOMAIN "python-discord.workers.dev"
```

Comment:
Why not use a domain under `pythondiscord.com`?

Reply:
I don't have any control over where the worker is deployed.